airflow/kubernetes/pod_generator.py

```diff
@@ -116,6 +116,8 @@ class PodGenerator:
     :type affinity: Optional[dict]
     :param hostnetwork: If True enable host networking on the pod
     :type hostnetwork: bool
+    :param host_aliases: A list of hostAliases.
+    :type host_aliases: list hostAliases
```

Comment:
```suggestion
    :type host_aliases: list
```

Plus somewhere (not the type) a link to what structure this should have.